sdk/analytics/src/main/java/com/klaviyo/analytics/DeviceProperties.kt

```diff
@@ -62,16 +60,7 @@ internal object DeviceProperties {
     }
 
     val notificationPermission: Boolean
-        get() {
-            // Notification permissions only expected from Android 13 onwards
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
-                return ActivityCompat.checkSelfPermission(
-                    Registry.config.applicationContext,
-                    Manifest.permission.POST_NOTIFICATIONS
-                ) == PackageManager.PERMISSION_GRANTED
-            }
-            return true
-        }
+        get() = NotificationManagerCompat.from(Registry.config.applicationContext).areNotificationsEnabled()
```

Comment:
this was the same property that caused the permission error last summer, sure wish I'd discovered this method then.